diff --git a/load-plugins.js b/load-plugins.js
index d8e779cc81380cc26bb37846ba025b93150f888a..8a74548c075a8459ba984a50a63c59e5ccbd67d1 100644
--- a/load-plugins.js
+++ b/load-plugins.js
@@ -9,7 +9,7 @@ class Plugins {
   }
 
   has(type) {
-    return this.list.some(i => i.name === `@size-limit/${type}`)
+    return this.list.some(i => i.name === `@size-limit/${type}` || i.name === `size-limit-${type}`)
   }
 }
 
@@ -19,7 +19,7 @@ module.exports = function loadPlugins(pkg) {
   let list = toArray(pkg.packageJson.dependencies)
     .concat(toArray(pkg.packageJson.devDependencies))
     .concat(toArray(pkg.packageJson.optionalDependencies))
-    .filter(i => i.startsWith('@size-limit/'))
+    .filter(i => i.startsWith('@size-limit/') || i.startsWith('size-limit-'))
     .reduce(
       (all, i) =>
         all.concat(